src/resources/team.yaml

```diff
@@ -62,3 +62,36 @@ actions:
     comment: |
       Returns the compact records for all users that are members of the team.
 
+  - name: addUser
+    class: users
+    method: POST
+    path: "/teams/%d/addUser"
+    params:
+      - name: team
+        <<: *Param.TeamId
+        required: true
+      - name: user
+        type: String
```

Comment:
we have a parameter type that is User, don't we? we should use that. I think we actually have `Param.UserId`, but given that we accept email addresses (or `me`) in any scenario, we should use that and update its description to indicate that it can take on those values.